photon-core/src/main/java/org/photonvision/common/configuration/NetworkConfig.java

```diff
@@ -74,18 +74,18 @@ public NetworkConfig(
             @JsonProperty("runNTServer") boolean runNTServer,
             @JsonProperty("shouldManage") boolean shouldManage,
             @JsonProperty("shouldPublishProto") boolean shouldPublishProto,
-            @JsonProperty("networkManagerIface") String networkManagerIface,
+            @JsonProperty("networkManagerInterface") String networkManagerInterface,
             @JsonProperty("setStaticCommand") String setStaticCommand,
-            @JsonProperty("setDHCPcommand") String setDHCPcommand) {
+            @JsonProperty("setDHCPCommand") String setDHCPCommand) {
```

Comment:
Why chance these? It'll brick people's network configs if they try upgrading, potentially quietly.

Reply:
fair enough, undo, just weird naming imo.